### PR TITLE
updated schema database name

### PR DESCRIPTION
**Pull request by gkpeel**
The only change I made was to models/schema.sql from the example_db names to 'begster' so it matches the config.json file

-- Drops the blogger if it exists currently --
DROP DATABASE IF EXISTS begslist;
-- Creates the "blogger" database --
CREATE DATABASE begslist;

Also changed config.json to match my personal Workbench password, not sure if this file should be .gitignored for everyone or else might keep overwriting one another's pw with every commit. 

Basically wanted to use this as a Slack test. Will work on creating the four models next.

### CODE REVIEW

**Review by alexszeliga — APPROVED** [review]
This seems appropriate.
